### PR TITLE
set role for admin page

### PR DESCRIPTION
**Pull request by duyanh1992**

### CODE REVIEW

File: src/components/Admin/Menu.js [hunk 1]
```diff
@@ -51,7 +54,7 @@ class Menu extends Component {
             
             <div className="dropdown">
               <a className="nav-link dropdown-toggle" href="/a" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                Welcome, User !
+                Welcome, <span style={{ color: 'red' }}>{userName}</span> !
```
Comment (AnhDuy3392) [inline src/components/Admin/Menu.js:57]: User Name is styled color red anytime?

File: src/components/MainSite/common/SearchBar.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ export default class SearchBar extends Component {
                 </div>
                 <div className="dropdown mr-3">
                     <a className="nav-link dropdown-toggle text-secondary font-weight-bold" href="#a" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        {userContent}
+                        Welcome <span style={{ color: 'red' }}>{userContent}</span>
```
Comment (AnhDuy3392) [inline src/components/MainSite/common/SearchBar.js:68]: User Name is styled color red anytime?

File: src/components/Admin/Menu.js (hunk 1 above)
Comment (duyanh1992) [reply src/components/Admin/Menu.js:57]: Yes. We can only access to admin page after logged in by admin acccount. So,only admin name is showed with red color

File: src/components/MainSite/common/SearchBar.js (hunk 2 above)
Comment (duyanh1992) [reply src/components/MainSite/common/SearchBar.js:68]: Yeah. I'll fix it

**Pull request merged**